triplets/community/arm64-android.cmake

```diff
@@ -3,3 +3,5 @@ set(VCPKG_CRT_LINKAGE static)
 set(VCPKG_LIBRARY_LINKAGE static)
 set(VCPKG_CMAKE_SYSTEM_NAME Android)
 set(VCPKG_BUILD_TYPE release)
+set(VCPKG_C_FLAGS "-O2 -DNDEBUG")
+set(VCPKG_CXX_FLAGS "-O2 -DNDEBUG -Wno-unused-command-line-argument -Wl,--hash-style=both -fno-stack-protector -fno-unwind-tables -fno-asynchronous-unwind-tables -frtti -ffast-math -ffp-contract=fast -fvisibility-inlines-hidden -fvisibility=hidden -fomit-frame-pointer -ffunction-sections -fdata-sections -Wl,--gc-sections -Wl,-Bsymbolic-functions")

```

Comment:
I don't think exceptions will still work with the `-fno-unwind-tables` option.

Reply:
fixed, removed those for the android shared lib too